_layouts/api.html

```diff
@@ -49,6 +49,9 @@ <h2>
   {% endif %}
   {% if page.customisation contains 'Adaptive' %}
     The API also supports <a href="/adaptive">adaptive</a> customisation.
+  {% endif %} <br>
+  {% if page.file_translation %}
+    The API also offers <a href='/file-translation'>file translation</a>.
```

Comment:
I prefer "supports", because "offers" is more like marketing.

Reply:
But ideally we'd have a list of type and just put it in the infobox:

> File types: ...

Reply:
> But ideally we'd have a list of type and just put it in the infobox:
> 
> > File types: ...

We can link this to the issue that needs to be created to include the file types instead of giving a boolean value for file translation in the `apis.json`.

Reply:
Here's the issue #613